framework/web/XmlResponseFormatter.php

```diff
@@ -50,7 +50,11 @@ class XmlResponseFormatter extends Component implements ResponseFormatterInterfa
      * @since 2.0.7
      */
     public $useTraversableAsArray = true;
-
+    /**
+     * @var bool if object tags should be added
+     * @since 2.0.11
+     */
+    public $objectTags = true;
```

Comment:
`objectTags` does not sound appropriate for a boolean field.
Possible names could be:
- `createObjectTags`
- `useObjectTags`
- `addObjectTags`